crates/cdk-integration-tests/tests/test_fees.rs

```diff
@@ -92,7 +92,7 @@ async fn test_fake_melt_change_in_quote() {
             mint_quote.clone(),
             SplitTarget::default(),
             None,
-            tokio::time::Duration::from_secs(15),
+            tokio::time::Duration::from_secs(60),
```

Comment:
I don't mind setting these higher if they solve our ci issues but I think its likely covering up an issue more then solving it, so we should try to solve it for real in another pr.